_docs/getting-started.md

```diff
@@ -126,14 +126,14 @@ A `node` is a unique string, which is Bistro's way of making tasks and
 tracking resources.  To make a task, you need:
```

Comment:
Might not be important: Github's code review highlights this entire block in bright red. Does this mean there is a Markdown syntax error in this file?